registry/txt_test.go

```diff
@@ -977,6 +1002,7 @@ func testTXTRegistryMissingRecordsWithPrefix(t *testing.T) {
 		},
 		{
 			DNSName:    "txt.a-oldformat2.test-zone.example.org",
+			SetIdentifier: "one",
```

Comment:
Aren't we supposed to see the third missing new format TXT record for the identifier `two`?

Reply:
No, because the `two` was owned by another owner. Refactored now.